gulp/tasks/build.js

```diff
@@ -182,13 +169,24 @@ function compileIndvES2015ModulesToBrowser() {
         }
       }),
       new webpack.optimize.UglifyJsPlugin({
-        sourceMap: true
+        sourceMap: true,
+        mangle: {
+          props: {
+            ignore_quoted: true,
+            regex: /_$/,
```

Comment:
Nice! The SDK code only uses `_` as a suffix but other imported dependencies must be using it as a prefix too. I'll update it with this change.